grpc/grpc.go

```diff
@@ -56,11 +46,8 @@ func Commit(client ipc.ExecutionEngineServiceClient,
 	prestateHash []byte,
 	effects []*transforms.TransformEntry,
 	protocolVersion *state.ProtocolVersion) (postStateHash []byte, validators []*ipc.Bond, errMessage string) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
-	defer cancel()
-
 	r, err := client.Commit(
-		ctx,
+		context.TODO(),
 		&ipc.CommitRequest{
 			PrestateHash:    prestateHash,
 			Effects:         effects,
```

Comment:
Some out-updated code and seems-to-be-tiny but,..

In error string concat, I recommend to use `fmt.Sprintf()` rather than '+' operator

Reply:
https://github.com/hdac-io/casperlabs-ee-grpc-go-util/pull/7/commits/5c3f83e7a1cdb63b64d710e1f8f19873d9f8a871